components/Button.tsx

```diff
@@ -27,123 +27,123 @@ import defaultTheme from './theme';
 import { Spinner } from './theme/icons';
 
 export const UnStyledButton = styled('button')`
-  background: transparent;
```

Comment:
We need to decide on tabs vs spaces and indent standard. The changes here are simply whitespace.

Reply:
Let's leave it with what makes the linter happy. :)